samples/cpp/speculative_decoding_lm/speculative_decoding_lm.cpp

```diff
@@ -116,34 +30,25 @@ int main(int argc, char* argv[]) try {
     };
 
     ov::genai::SchedulerConfig scheduler_config;
-    // batch size
-    scheduler_config.max_num_batched_tokens = use_prefix ? 256 : 32;
-    // cache params
-    scheduler_config.num_kv_blocks = 364;
+    scheduler_config.cache_size = 5;
     scheduler_config.block_size = get_default_block_size(device);
-    // mode - vLLM or dynamic_split_fuse
-    scheduler_config.dynamic_split_fuse = dynamic_split_fuse;
-    // vLLM specific params
-    scheduler_config.max_num_seqs = 2;
-    scheduler_config.enable_prefix_caching = use_prefix;
 
     // It's possible to construct a Tokenizer from a different path.
     // If the Tokenizer isn't specified, it's loaded from the same folder.
-    ov::genai::LLMPipeline pipe(model_path, device, ov::genai::draft_model(draft_model_path, device), ov::genai::scheduler_config(scheduler_config));
-
-    if (use_prefix) {
-        std::cout << "Running inference for prefix to compute the shared prompt's KV cache..." << std::endl;
-        auto generation_results = pipe.generate(prefix_str, ov::genai::greedy());
-    }
+    // User can run main and draft model on different devices.
+    // Please, set device for main model in `LLMPipeline` constructor and in in `ov::genai::draft_model` for draft.
+    // Example to run main_model on GPU and draft_model on CPU:
+    // ov::genai::LLMPipeline pipe(main_model_path, "GPU", ov::genai::draft_model(draft_model_path, "CPU"), ov::genai::scheduler_config(scheduler_config));
```

Comment:
maybe we can define 2 variables `main_model_device` and `draft_model_device` ? by default they are both CPU, but you can write that powerful GPU owners can change main device to GPU